cmake/symlink_install/catkin_symlink_install.cmake.in

```diff
@@ -11,7 +11,7 @@ file(WRITE "${CMAKE_CURRENT_BINARY_DIR}/symlink_install_manifest.txt")
 # :type ARGN: various
 #
 function(catkin_symlink_install_directory)
-  cmake_parse_arguments(ARG "OPTIONAL" "DESTINATION" "DIRECTORY;PATTERN;PATTERN_EXCLUDE" ${ARGN})
+  cmake_parse_arguments(ARG "OPTIONAL" "DESTINATION;CURRENT_SOURCE_DIR" "DIRECTORY;PATTERN;PATTERN_EXCLUDE" ${ARGN})
```

Comment:
Please document the new argument in the doc block.

Nitpick: please insert it before *DESTINATION* to follow alphabetical order.

Same below.

Reply:
Documented new argument and put it in alphabetical order 542ec01d1f0399f31f6111181de03324eb091ce2